src/components/FormField/styles.js

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+import { themeGet } from "@styled-system/theme-get";
+import { Field } from "formik";
+
+export const ErrorMessageText = styled.small`
+  font-size: 14px;
+  color: ${themeGet("colors.error")};
+  margin-top: 5px;
+  display: block;
+`;
+
+export const Hakl = styled(Field)`
+  display: flex;
+  font-size: 18px;
+  padding: 10px 20px 10px 20px;
```

Comment:
a shorter version that would produce the same result would be `padding: 10px 20px`
(i think I wrote this tho lol)